ext/random/tests/03_randomizer/methods/getFloat_error.phpt

```diff
@@ -73,10 +73,17 @@ foreach ([
     } catch (ValueError $e) {
         echo $e->getMessage(), PHP_EOL;
     }
+
+    try {
+        // There is no float between the two parameters, thus making the OpenOpen interval empty.
+        var_dump(randomizer()->getFloat(1.0, 1.0000000000000002, $boundary));
```

Comment:
I think using `1 + PHP_FLOAT_EPSILON` instead of the hard-coded value would clarify this a bit:
```suggestion
        var_dump(randomizer()->getFloat(1.0, 1 + PHP_FLOAT_EPSILON, $boundary));
```

Reply:
Ah, indeed. The epsilon is well-defined here. I've used `nextafter()` in a stand-alone C program to determine the value.